cmd/vulkan_sample/main.cpp

```diff
@@ -749,6 +760,9 @@ int main(int argc, const char** argv) {
   VkFence ready_fences[kBufferingCount];
   VkSemaphore swapchain_image_ready_semaphores[kBufferingCount];
   VkSemaphore render_done_semaphores[kBufferingCount];
+#if defined(__ANDROID__)
+  VkSemaphore frame_boundary_done_semaphores[kBufferingCount];
+#endif
```

Comment:
Why the extra semaphore? We could just use `render_done_semaphore`.

Reply:
Because `render_done_semaphore` wait is already done on vkQueuePresent (and binary semaphores cannot have more than one wait per signal).